src/operator/contrib/ctc_include/LICENSE

```diff
@@ -188,7 +186,7 @@ Copyright 2015-2016 Baidu USA LLC.  All rights reserved.
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2015-2016, Baidu USA LLC.
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
year and copyright missing?

Reply:
The comment here "A number of Apache licenses have incorrect appendixes with the copyright line filled in e.g. [2]" indicates that it should not be.